src/mono/monoaotcross.proj

```diff
@@ -43,20 +43,23 @@
       <MonoAotTargetArchitecture>$(MonoAotTargetRid.Substring($([MSBuild]::Add(1, $(MonoAotTargetRid.LastIndexOf('-'))))))</MonoAotTargetArchitecture>
       <MonoAotHostOS>$(TargetOS)</MonoAotHostOS>
       <MonoAotHostOS Condition="'$(TargetsLinuxMusl)' == 'true'">linux-musl</MonoAotHostOS>
+      <MonoAotHostArchitecture>$(TargetArchitecture)</MonoAotHostArchitecture>
```

Comment:
Doesn't this override L43 unconditionally? Is that the intent?

Reply:
this is different, host vs target

Reply:
Ok got it. Shouldn't the host architecture default to the target architecture? That's at least what we do in the VMR: https://github.com/dotnet/sdk/blob/dcaddf212a5254439ba4fac8b5efc1bf56c2b52b/src/SourceBuild/content/Directory.Build.props#L48-L49

Reply:
Oh I see. These properties are being used differently here. Interesting.